langchain/embeddings/mlflow_gateway.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Any, Iterator, List, Optional
+
+from pydantic import BaseModel
+
+from langchain.embeddings.base import Embeddings
+
+
+def chunk(texts: List[str], size: int) -> Iterator[List[str]]:
```

Comment:
should this be internal?

```suggestion
def _chunk(texts: List[str], size: int) -> Iterator[List[str]]:
```

Reply:
Updated!